scripts/update-version.sh

```diff
@@ -23,78 +19,44 @@ eval projects=( "cli"
 
 cd `dirname $0`
 
-prefix="@alfresco\/adf-"
-
-projectslength=${#projects[@]}
-
 show_help() {
     echo "Usage: update-version.sh"
     echo ""
     echo "-vj or -versionjsapi  to use a different version of js-api"
     echo "-v or -version  version to update"
-    echo "-major increase the major number and reset minor and patch"
-    echo "-minor increase the minor number and reset the patch number"
-    echo "-patch increase the patch number"
-    echo "-nextalpha update next alpha version of js-api and lib automatically"
-    echo "-nextbeta update next beta version of js-api and lib automatically"
-    echo "-alpha update last alpha version of js-api and lib automatically"
-    echo "-beta update beta alpha version of js-api and lib automatically"
     echo "-gnu for gnu"
 }
 
-last_alpha_mode() {
-    length=`expr $projectslength - 1`
-    echo "====== Auto find last ALPHA version of ${projects[${length}]} ====="
-    VERSION=$(npm view @alfresco/adf-${projects[${length}]}@alpha version)
-
-    echo "====== version lib ${VERSION} ====="
-}
+get_next_version() {
+    PKG_VERSION=`node -p "require('$1/package.json')".version;`;
```

Comment:
Is the ```;`;``` a typo or valid syntax?

Reply:
first one is part of the node.js script executed